### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by elmejorlui**
finished the components for the login and home pages

this PR completes the issues

#2 
#3 
#4 
#5 
#6 
#7 
#9

### CODE REVIEW

File: packages/authentication/src/components/Login/index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  Input,
+  InputPassword,
+  LinkButton,
+} from '@siiges-ui/shared';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { Typography } from '@mui/material';
+import Link from 'next/link';
+
+export default function SignIn() {
```
Comment (csiges) [inline packages/authentication/src/components/Login/index.jsx:13]: Try to rename the component

File: packages/shared/src/components/Button.jsx [hunk 2]
```diff
@@ -1,5 +1,50 @@
 import React from 'react';
+import Button from '@mui/material/Button';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
 
-export default function Button() {
-  return <button type="submit">TEST BUTTON</button>;
+function ButtonLogin({
+  type,
+  color,
```
Comment (csiges) [inline packages/shared/src/components/Button.jsx:8]: Try to rename the names of the component equal to the filename in UpperCamelCase, and other best practice is put a folder with the name of the main component, and a file index inside the component.

File: packages/shared/__tests__/button.test.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
```
Comment (csiges) [inline packages/shared/__tests__/button.test.jsx:2]: Try to put your tests in the same way as your architecture, for example (__tests__/src/components/ButtonLogin.test.jsx)
This is for a better organazed project.

File: apps/siiges-app/pages/UserProfile/index.jsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                <Image
+                  alt="avatar"
+                  src="/avatar.png"
+                  quality={100}
+                  width="300px"
+                  height="300px"
+                  style={{
+                    zIndex: 1,
+                    overflow: 'hidden',
+                  }}
```
Comment (hecttoressau) [inline apps/siiges-app/pages/UserProfile/index.jsx:42]: You can easily make this part a component and use it here
```suggestion
// file new-component.js
export default function NewComponent(src) {
  return (
  <Image
     alt="avatar"
     src={src}
     quality={100}
     width="300px"
     height="300px"
     style={{
     zIndex: 1,
     overflow: 'hidden',
     }}
   />
  )
 }
 // userProfile/index.js
 <Grid item xs={4} sx={{ marginTop: 7 }}>
   <NewComponent src='/avata.png'
```

File: apps/siiges-app/pages/UserProfile/index.jsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                >
+                  <Typography variant="p">
+                    Luis Manuel de Alba Villaseñor
+                  </Typography>
+                  <br />
+                  <Divider sx={{ marginY: 1 }} />
+                  <Typography variant="p">
+                    El jefe de jefes
+                  </Typography>
+                </Paper>
```
Comment (hecttoressau) [inline apps/siiges-app/pages/UserProfile/index.jsx:60]: We could create a new component with this
```suggestion
// new-component.js
exports default NewComponent(name, rol) {
        return (
                <Paper
                  sx={{
                    padding: 2,
                    marginTop: 3,
                    width: 300,
                    textAlign: 'center',
                  }}
                >
                  <Typography variant="p">
                    {name}
                  </Typography>
                  <br />
                  <Divider sx={{ marginY: 1 }} />
                  <Typography variant="p">
                    {rol}
                  </Typography>
                </Paper>
        );
}
// UserProfile/index.jsx
<NewComponent name="Luis Manuel de Alba Villaseñor" rol="El jefe de jefes"
```

File: apps/siiges-app/pages/UserProfile/index.jsx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+              </Grid>
+              <Grid item xs={8}>
+                <Typography variant="h5" gutterBottom component="div">
+                  Información Personal
+                </Typography>
+                <Divider sx={{ bgcolor: 'orange', marginBottom: 5 }} />
+                <Grid container spacing={5}>
+                  <Grid item xs={4}>
+                    <Input label="Nombre(s)" id="name" name="name" auto="name" size="small" />
```
Comment (hecttoressau) [inline apps/siiges-app/pages/UserProfile/index.jsx:69]: We can create a new component with this
```suggestion
exports default InputSamll( label, id, name, auto) {
  <Input label={label} id={id} name={name }auto={auto} size="small" />
}
```

File: apps/siiges-app/pages/UserProfile/index.jsx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+                  </Grid>
+                  <Grid item xs={4}>
+                    <Input label="Primer Apellido" id="lastname1" name="lastname1" auto="lastname1" size="small" />
+                  </Grid>
+                  <Grid item xs={4}>
+                    <Input label="Segundo Apellido" id="lastname2" name="lastname2" auto="lastname2" size="small" />
+                  </Grid>
```
Comment (hecttoressau) [inline apps/siiges-app/pages/UserProfile/index.jsx:76]: We can create a new component from this
```suggestion
//gri-itemt.js
exports default GridItem({xs, InputSmallProps }) {
  return (
    <InputSmall ...>
<Grid item xs={4}>
<Input label="Segundo Apellido" id="lastname2" name="lastname2" auto="lastname2" size="small" />
</Grid>
```

File: apps/siiges-app/pages/home/index.jsx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+        alt="travel"
+        src="/Fondo.jpg"
+        layout="fill"
+        objectFit="cover"
+        quality={100}
+        style={{
+          zIndex: -1,
+          overflow: 'hidden',
+        }}
+      />
```
Comment (hecttoressau) [inline apps/siiges-app/pages/home/index.jsx:24]:
```suggestion
// file new-component.js
export default function NewComponent(src) {
  return (
   <Image
        alt="travel"
        src={src}
        layout="fill"
        objectFit="cover"
        quality={100}
        style={{
          zIndex: -1,
          overflow: 'hidden',
        }}
      />
  )
 }
 // home/index.js
 <Grid item xs={4} sx={{ marginTop: 7 }}>
   <NewComponent src='/avata.png'
```
**Note:** try to abstract ti more in order to make it more reusable

File: apps/siiges-app/pages/home/index.jsx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+                display: 'flex',
+                '& > :not(style)': {
+                  mt: 5,
+                  mx: 1,
+                  width: 350,
+                  height: 250,
+                },
+              }}
+            >
+              <PaperHome
```
Comment (hecttoressau) [inline apps/siiges-app/pages/home/index.jsx:58]: Rename it to make it more expresive

File: apps/siiges-app/pages/login/index.jsx [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+        alt="travel"
+        src="/Fondo.jpg"
+        layout="fill"
+        objectFit="cover"
+        quality={100}
+        style={{
+          zIndex: -1,
+          overflow: 'hidden',
+        }}
+      />
```
Comment (hecttoressau) [inline apps/siiges-app/pages/login/index.jsx:22]: Make this a component, try to make it abstract to use it multiple times

File: apps/siiges-app/pages/password/index.jsx [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+import { RecoverPass } from '@siiges-ui/authentication';
+import { Logo, Header, Navbar } from '@siiges-ui/shared';
+import { Grid } from '@mui/material';
+import { ThemeProvider } from '@mui/material/styles';
+import theme from '../theme';
+
+function Password() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Image
```
Comment (hecttoressau) [inline apps/siiges-app/pages/password/index.jsx:12]: Make this a component, try to make it abstract to use it multiple times

File: apps/siiges-app/pages/signup/index.jsx [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+import { Register } from '@siiges-ui/authentication';
+import { Logo, Header, Navbar } from '@siiges-ui/shared';
+import { Grid } from '@mui/material';
+import { ThemeProvider } from '@mui/material/styles';
+import theme from '../theme';
+
+function SignUpPage() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Image
```
Comment (hecttoressau) [inline apps/siiges-app/pages/signup/index.jsx:12]: Make this a component, try to make it abstract to use it multiple times

**Review by hecttoressau — APPROVED** [review]
LGTM

**Pull request merged**